src/fu-tool.c

```diff
@@ -976,15 +982,25 @@ fu_util_update_all (FuUtilPrivate *priv, GError **error)
 		if (!fu_util_is_interesting_device (dev))
 			continue;
 		/* only show stuff that has metadata available */
-		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_SUPPORTED))
+		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE))
+			continue;
+		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_SUPPORTED)) {
+			/* TRANSLATORS: message letting the user know no device upgrade available due to missing on LVFS
+			* %1 is the device name */
+			g_printerr (_("• %s does not have firmware updates posted on any enabled remotes\n"), fwupd_device_get_name (dev));
```

Comment:
I don't like `posted` and `remotes` makes it all sound very technical. Maybe just `%s has no available firmware updates`?

Reply:
Yeah that's good.

Reply:
Adjusted